src/net/p2p.rs

```diff
@@ -1193,22 +1196,45 @@ impl PeerNetwork {
         Ok(())
     }
 
-    /// Check to see if we can register a peer with a given public key
+    /// Check to see if we can register a peer with a given public key in a given direction
     pub fn can_register_peer_with_pubkey(
         &mut self,
         nk: &NeighborKey,
         outbound: bool,
         pubkh: &Hash160,
     ) -> Result<(), net_error> {
+        // can't talk to myself
+        let my_pubkey_hash = Hash160::from_node_public_key(&Secp256k1PublicKey::from_private(
+            &self.local_peer.private_key,
+        ));
+        debug!(
+            "{:?}: my public key hash is {}",
```

Comment:
this might be a bit much.  I had left the master process running since last night, and it is present `2_060_652` times in the log (out of a total `15_184_304` lines)

Reply:
Ha, yeah -- will remove these.  This is an artifact from trying to debug #1978.  There are a few other lines that will need removing.